examples/browser-vue/src/plugins/vue-ipfs.js

```diff
@@ -0,0 +1,22 @@
+import IPFS from 'ipfs'
+
+function promisifyIpfs(opts) {
+  return new Promise((resolve, reject) => {
+    const node = new IPFS(opts)
+    node.once('ready', () => resolve(node))
+    node.once('error', err => reject(err))
+  })
```

Comment:
With `0.37.0-rc.0` (the upcoming release) you do not need to do this any more.

You can just `await IPFS.create(opts)`. Can we use this new method in this example?